src/gen_apple/ga_dev_gps.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ga_dev_gps.h"
+
+#define GPSECHO  false //Define true to echo gps data to serial monitor
+
+SoftwareSerial mySerial(_PIN_GPS_TX, _PIN_GPS_RX);
+Adafruit_GPS GPS(&mySerial);
+
+void ga_dev_gps_open(void) {
+
+    pinMode(_PIN_GPS_ENABLE , OUTPUT);
+    //Enable pin needs to be pulled low;
+    digitalWrite(_PIN_GPS_ENABLE , HIGH);
+
+    //GPS baud rate should be 9600
+    GPS.begin(9600);
+
+    //RMC = Recommended Minimum Data for GPS
+    //GGA = Fix Informtation
+    GPS.sendCommand(PMTK_SET_NMEA_OUTPUT_RMCGGA);
+
+    //Update at 1Hz, Don't use anything higher to allow parsing
+    GPS.sendCommand(PMTK_SET_NMEA_UPDATE_1HZ);
+
+}
+
+void ga_dev_gps_high(void){
```

Comment:
You could probably combine the `gps_high` and `gps_low` function into one function that is passed an argument to determine what to change the state to. High and low might be vague terms; you can probably call the function something like ga_dev_gps_power and pass it defined macros called OFF, or ON. If you can think of a better name for the function or macros try that and see how it looks.